app/views/insured/families/verification.html.erb

```diff
@@ -11,12 +11,20 @@
       <%= render partial: "insured/families/verification/documents_verification_list" %>
     </div>
   </div>
-  <% if @person && @family_members.present? && individual_market_is_enabled? %>
-    <%= render 'insured/families/verification/verification' %>
-  <% end %>
 
-  <% if EnrollRegistry.feature_enabled?(:financial_assistance) %>
-    <%= render partial: "financial_assistance/applications/verifications/verification", locals: {family_id: @family.id} %>
+  <% if EnrollRegistry.feature_enabled?(:show_new_verifications_household_summary) %>
+    <% if @person && @family_members.present? && individual_market_is_enabled? %>
```

Comment:
nitpick, but this is the same check as below. think it makes more sense to move the flag check inside of it